solr/core/src/java/org/apache/solr/response/transform/ChildDocTransformerFactory.java

```diff
@@ -61,6 +57,13 @@
  */
 public class ChildDocTransformerFactory extends TransformerFactory {
 
+  public static final String PATH_SEP_CHAR = "/";
```

Comment:
I don't see why these are declared as Strings and not `char`.  It's clumsy later to do `charAt(0)`